domains/transfer/Transfer.API/API/Cvr/Api/v2024_01_03/Controllers/CvrController.cs

```diff
@@ -0,0 +1,100 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using API.Cvr.Api.Clients.Cvr;
+using API.Cvr.Api.Models;
+using API.Cvr.Api.v2024_01_03.Dto.Responses;
+using Asp.Versioning;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Cvr.Api.v2024_01_03.Controllers;
+
+[Authorize]
+[ApiController]
+[ApiVersion(20240103)]
+[Route("api/cvr")]
+public class CvrController(CvrClient client) : Controller
+{
+    /// <summary>
+    /// Get CVR registered company information for multiple CVR numbers
+    /// </summary>
+    /// <response code="200">Successful operation</response>
+    /// <response code="400">A non-empty request body is required.</response>
+    [ProducesResponseType(typeof(CvrCompanyListResponse), 200)]
+    [ProducesResponseType(typeof(void), 400)]
+    [HttpPost]
+    public async Task<ActionResult<CvrCompanyListResponse>> GetCvrCompanies([FromBody] List<string> cvrNumbers)
```

Comment:
the body should also be in a dto according to our guidelines

Reply:
Created dto, and i am using it:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/f069c46be91fcec251339ce4157db8132dbc5c55